src/App.js

```diff
@@ -1,27 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Calculator />;
 }
```

Comment:
- [ ] Please know that the App.js component should be class-based, kindly fix this.

Reply:
> * [ ]  Please know that the App.js component should be class-based, kindly fix this.

Let me fix this. Thanks for the advice